keyboards/kinesis/nguyenvietyen/rules.mk

```diff
@@ -0,0 +1,13 @@
+CUSTOM_MATRIX = lite
+
```

Comment:
```suggestion
```

Reply:
According to https://beta.docs.qmk.fm/using-qmk/hardware-features/custom_matrix, this is needed. Is the doc perhaps incorrect?

Reply:
Oh wait, I missed now. Nvm.